libs/dh/dev-examples/feature-processes/src/details/gridareas.component.ts

```diff
@@ -0,0 +1,70 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { ChangeDetectionStrategy, Component, effect, input, viewChild } from '@angular/core';
+import { MatSort, MatSortModule } from '@angular/material/sort';
+import { TranslocoDirective } from '@ngneat/transloco';
+
+import { WATT_CARD } from '@energinet-datahub/watt/card';
+import { WattDataTableComponent } from '@energinet-datahub/watt/data';
+import { WattTableDataSource, WattTableColumnDef, WATT_TABLE } from '@energinet-datahub/watt/table';
+
+import { DhProcessCalculationGridArea } from '../types';
+
+@Component({
+  imports: [MatSortModule, TranslocoDirective, WATT_TABLE, WattDataTableComponent, WATT_CARD],
+  selector: 'dh-calculation-details-grid-areas',
```

Comment:
issue: this selector doesn't look right

Reply:
Why not? It is grid areas only for calculations?

Reply:
I didn't realize this was only for calculations, so I see your point. That said, do we want this component to use the same selector/component "prefix" as components in `wholesale/feature-calculations`? Again, normally I probably wouldn't care that much, but since this is about best practices, maybe we should settle on how we prefix components?

For example, there is the super-overly-long-but-technically-correct-way:
`dh-dev-examples-feature-processes-calculation-details-grid-areas`

Or the one that skips domain and the feature part (fx. we use this in calculations and actors):
`dh-processes-calculations-details-grid-areas`

Or the one like yours that just don't care because it is never used outside the feature (which is a valid argument):
`dh-calculations-details-grid-areas`

Reply:
What was my thought. That it was okay because it was never exported and only used as a feature component, so it can not be referenced in other domain / feature. What say you @dzhavat

Reply:
We have started naming components with shorter names when they are internal for a library. From that point of view it doesn't matter much that the selector is not technically correct.

Looking more long-term, the Angular team is looking at **selectorless** components, which will land at some point this year. If **selectorless** means what we think it does, the selector itself will become less important over time.